<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AlbumCreateDialog</class>
 <widget class="QDialog" name="AlbumCreateDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>643</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create Album</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="HeaderFrame" name="headerFrame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="mainPage">
      <layout class="QGridLayout" name="mainPageGLayout">
       <item row="0" column="0">
        <widget class="QFrame" name="frame">
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <layout class="QGridLayout" name="frameGLayout">
          <property name="margin">
           <number>0</number>
          </property>
          <property name="spacing">
           <number>6</number>
          </property>
          <item row="0" column="0">
           <layout class="QGridLayout" name="headerGLayout">
            <property name="margin">
             <number>3</number>
            </property>
            <item row="0" column="0">
             <widget class="QLabel" name="label_Title">
              <property name="text">
               <string>Album Name:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="lineEdit_Title"/>
            </item>
            <item row="0" column="2" rowspan="2">
             <widget class="QToolButton" name="AlbumThumbNail">
              <property name="minimumSize">
               <size>
                <width>64</width>
                <height>64</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>64</width>
                <height>64</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">
border: 2px solid white;
border-radius: 10px;
</string>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="Photo_images.qrc">
                <normaloff>:/images/album_64.png</normaloff>:/images/album_64.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>64</width>
                <height>64</height>
               </size>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_Category">
              <property name="text">
               <string>Category:</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QComboBox" name="comboBox_Category">
              <item>
               <property name="text">
                <string>Animals</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Family</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Friends</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Flowers</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Holiday</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Landscapes</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Pets</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Portraits</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Travel</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Work</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Random</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="label_Caption">
              <property name="text">
               <string>Caption:</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1" colspan="2">
             <widget class="QLineEdit" name="lineEdit_Caption"/>
            </item>
            <item row="4" column="0">
             <widget class="QLabel" name="label_Where">
              <property name="text">
               <string>Where:</string>
              </property>
             </widget>
            </item>
            <item row="4" column="1" colspan="2">
             <widget class="QLineEdit" name="lineEdit_Where"/>
            </item>
            <item row="5" column="0">
             <widget class="QLabel" name="label_Photographer">
              <property name="text">
               <string>Photographer:</string>
              </property>
             </widget>
            </item>
            <item row="5" column="1" colspan="2">
             <widget class="QLineEdit" name="lineEdit_Photographer"/>
            </item>
            <item row="6" column="0">
             <widget class="QLabel" name="label_Description">
              <property name="text">
               <string>Description:</string>
              </property>
             </widget>
            </item>
            <item row="6" column="1" rowspan="2" colspan="2">
             <widget class="QTextEdit" name="textEdit_Description"/>
            </item>
            <item row="7" column="0">
             <spacer name="descrVSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="2" column="0" colspan="3">
             <widget class="Line" name="headerHLine">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="1" column="0">
           <widget class="Line" name="frameHLine">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QGroupBox" name="optionGBox">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
              <horstretch>1</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="title">
             <string>Share Options</string>
            </property>
            <layout class="QGridLayout" name="optionGBoxGLayout">
             <item row="0" column="0">
              <layout class="QVBoxLayout" name="optionLVLayout">
               <item>
                <widget class="QLabel" name="label_Privacy">
                 <property name="text">
                  <string>Policy:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_Quality">
                 <property name="text">
                  <string>Quality:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_Comments">
                 <property name="text">
                  <string>Comments:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_Id">
                 <property name="text">
                  <string>Identity:</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item row="0" column="1">
              <layout class="QVBoxLayout" name="optionRVLayout">
               <item>
                <widget class="QComboBox" name="comboBox_Privacy">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <item>
                  <property name="text">
                   <string>Public</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Restricted</string>
                  </property>
                 </item>
                </widget>
               </item>
               <item>
                <widget class="QComboBox" name="comboBox_Quality">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <item>
                  <property name="text">
                   <string>Resize Images (&lt; 1Mb)</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Resize Images (&lt; 10Mb)</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Send Original Images</string>
                  </property>
                 </item>
                </widget>
               </item>
               <item>
                <widget class="QComboBox" name="comboBox_Comments">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <item>
                  <property name="text">
                   <string>No Comments Allowed</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Authenticated Comments</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Any Comments Allowed</string>
                  </property>
                 </item>
                </widget>
               </item>
               <item>
                <widget class="QComboBox" name="comboBox_Id">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <item>
                  <property name="text">
                   <string>Publish with Identity</string>
                  </property>
                 </item>
                </widget>
               </item>
              </layout>
             </item>
             <item row="0" column="2">
              <spacer name="optionHSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>168</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="dragPage">
      <layout class="QGridLayout" name="dragPageGLayout">
       <item row="0" column="0">
        <layout class="QVBoxLayout" name="dragVLayout" stretch="0,10">
         <item>
          <widget class="QLabel" name="label_DragAndDrop">
           <property name="text">
            <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8pt; font-weight:600;&quot;&gt; Drag &amp;amp; Drop to insert pictures. Click on a picture to edit details below.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QScrollArea" name="scrollAreaPhotos">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="MinimumExpanding">
             <horstretch>0</horstretch>
             <verstretch>10</verstretch>
            </sizepolicy>
           </property>
           <property name="acceptDrops">
            <bool>true</bool>
           </property>
           <property name="horizontalScrollBarPolicy">
            <enum>Qt::ScrollBarAsNeeded</enum>
           </property>
           <property name="widgetResizable">
            <bool>true</bool>
           </property>
           <property name="alignment">
            <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
           </property>
           <widget class="PhotoDrop" name="scrollAreaWidgetContents">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>621</width>
              <height>454</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true">QWidget#scrollAreaWidgetContents{border: none;}</string>
            </property>
            <layout class="QGridLayout" name="scrollAreaWidgetContentsGLayout"/>
           </widget>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QFrame" name="buttonFrame">
     <layout class="QHBoxLayout" name="buttonFrameHLayout">
      <property name="margin">
       <number>9</number>
      </property>
      <item>
       <spacer name="buttonHSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>419</width>
          <height>18</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QDialogButtonBox" name="buttonBox">
        <property name="standardButtons">
         <set>QDialogButtonBox::Cancel</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="backButton">
        <property name="text">
         <string>Back</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="addphotosButton">
        <property name="text">
         <string>Add Photos</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="publishButton">
        <property name="text">
         <string>Publish Album</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>HeaderFrame</class>
   <extends>QFrame</extends>
   <header>gui/common/HeaderFrame.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PhotoDrop</class>
   <extends>QWidget</extends>
   <header>gui/PhotoShare/PhotoDrop.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="Photo_images.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AlbumCreateDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>353</x>
     <y>529</y>
    </hint>
    <hint type="destinationlabel">
     <x>321</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
